content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -223,16 +224,13 @@ export class CollapsibleSection extends React.PureComponent {
               >
                 {this.renderIcon()}
                 <FluentOrText message={title} />
-              </span>
-              <span
-                className="click-target"
-                role="button"
-                tabIndex="0"
-                onKeyPress={this.onKeyPress}
-                onClick={this.onHeaderClick}
-              >
                 {isCollapsible && (
                   <span
+                    data-l10n-id={
+                      collapsed
+                        ? "newtab-section-expand-section-label"
+                        : "newtab-section-collapse-section-label"
+                    }
```

Comment:
@AllegroFox Has this change been verified with screen reader i.e moving aria-label here instead of span with className click-target correctly reads heading and aria-label attached to child span e.g. when tabbed to TopSites as 'TopSites Collapse Section'

Reply:
That's what happens for me with VoiceOver - not sure about other screen readers.